README.md

```diff
@@ -20,6 +20,19 @@ Now that you you have Exported Tests set up locally, navigate to the project usi
 npm ci
 ```
 
+### Deployment
+
+We distribute Exported Tests as ES6 and ES5 modules for backward-compatibility support of various testing frameworks. Import the desired module type from the following locations:
+
+* ES6 - `index.js`
+* ES5 - `dist/index.js`
```

Comment:
@elizabethsjudd 
do we want to name this differently and put it in the root?

```suggestion
* ES5 - `es5.js`
```
or open to other naming or maybe the directory becomes `es5/index.js`? Not sure...